lib/matplotlib/image.py

```diff
@@ -1374,26 +1383,28 @@ def imsave(fname, arr, vmin=None, vmax=None, cmap=None, format=None,
     Parameters
     ----------
     fname : str or file-like
-        Path string to a filename, or a Python file-like object.
-        If *format* is *None* and *fname* is a string, the output
-        format is deduced from the extension of the filename.
+        The filename or a Python file-like object to store the image in.
```

Comment:
I wouldn't kill the info that the filename's extension is used to determine the output format here. People rarely read the complete set of argument docs to find it in the `format`.